### PR TITLE
Bump version to 0.12.0 and update changelog

### PR DESCRIPTION
**Pull request by rocodes**
## Status

Waiting for #2129  ; can review once that's merged and backported into `release/0.12.0`

## Description

Update changelog and version

## Test Plan

- [ ] Base is `release/0.12.0`
- [ ] Version is `0.12.0` (We are heading straight to apt-qa)
- [ ] Changelog looks good
- [ ] CI

## Checklist

If these changes modify code paths involving cryptography, the opening of files in VMs or network (via the RPC service) traffic, Qubes testing in the staging environment is required. For fine tuning of the graphical user interface, testing in any environment in Qubes is required. Please check as applicable:

 - [ ] I have tested these changes in the appropriate Qubes environment
 - [ ] I do not have an appropriate Qubes OS workstation set up (the reviewer will need to test these changes)
 - [x] These changes should not need testing in Qubes

If these changes add or remove files other than client code, the AppArmor profile may need to be updated. Please check as applicable:

 - [ ] I have updated the [AppArmor profile](https://github.com/freedomofpress/securedrop-client/blob/HEAD/files/usr.bin.securedrop-client)
 - [x] No update to the AppArmor profile is required for these changes
 - [ ] I don't know and would appreciate guidance

If these changes modify the database schema, you should include a database migration. Please check as applicable:

 - [ ] I have written a migration and upgraded a test database based on `main` and confirmed that the migration is [self-contained] and applies cleanly
 - [ ] I have written a migration but have not upgraded a test database based on `main` and would like the reviewer to do so
 - [ ] I need help writing a database migration
 - [x] No database schema changes are needed

[self-contained]: https://github.com/freedomofpress/securedrop-client#generating-and-running-database-migrations

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+securedrop-client (0.12.0) unstable; urgency=medium
+
+  * see changelog.md
+
+ -- SecureDrop Team <securedrop@freedom.press>  Thu, 18 Jul 2024 16:37:39 -0400
+
 securedrop-client (0.12.0~rc1) unstable; urgency=medium
```
Comment (legoktm) [inline debian/changelog:7]: Drop the RC changelog please :)

File: changelog.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 0.12.0
+
+* Support viewing .heic and .avif files (#2110)
+* Fix printing of Office documents requiring conversion to PDF (#2119)
+* Show error details on Error screen of Export Wizard (#)
```
Comment (legoktm) [inline changelog.md:7]: Missing the number

**Comment by rocodes** [comment]
Sorry, sloppy. fixed in https://github.com/freedomofpress/securedrop-client/commit/c350c75fcbaf57ba9472662598e003594351f0b5

File: changelog.md [hunk 3]
```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 0.12.0
+
+* Support viewing .heic and .avif files (#2110)
+* Fix printing of Office documents requiring conversion to PDF (#2119)
+* Show error details on Error screen of Export Wizard (#2098)
+* Improve Export Wizard handling of button presses (#1926)
+* Add hover state and animation to Print dialog buttons (#2125)
```
Comment (legoktm) [inline changelog.md:9]: Let's also add #2131?

```suggestion
* Add hover state and animation to Print dialog buttons (#2125)
* Upgrade `openssl` Rust crate to 0.10.66 for RUSTSEC-2024-0357 (#2131)
```

Comment (rocodes) [reply changelog.md:9]: addressed in 2ffb26d6

**Review by legoktm — APPROVED** [review]

**Pull request merged**